gluten-iceberg/src/test/scala/org/apache/gluten/execution/VeloxIcebergSuite.scala

```diff
@@ -58,8 +58,7 @@ class VeloxIcebergSuite extends WholeStageTransformerSuite {
     }
   }
 
-  // Ignored due to failures, see https://github.com/apache/incubator-gluten/issues/5362
-  ignore("iceberg bucketed join", Some("3.4")) {
+  testWithSpecifiedSparkVersion("iceberg bucketed join", Some("3.4")) {
```

Comment:
Should we enable these tests for 3.5 as well?

Reply:
Thanks for comments. The `Some("3.4")` means that this UT will be executed for Spark with version >= 3.4.